Makefile

```diff
@@ -75,6 +75,7 @@ export DOCKER_TAG=$(CAPELLA_VERSION)-$(CAPELLA_DOCKERIMAGES_REVISION)
 
 # Should be 'latest', the branch name, the commit hash or a Git tag name
 export CAPELLA_DOCKERIMAGES_REVISION ?= latest
+export JUPYTER_NOTEBOOK_REVISION ?= python-3.11
```

Comment:
I would suggest naming this `JUPYTER_VERSION` as it is not really a revision and is mostly similar to our `CAPELLA_VERSION`. If you look into the `image-builder.yml` you can also see a potential use case and how we could use the `JUPYTER_VERSION` there.